### ISSUE TITLE
[HOLD for payment 2024-03-29] [$500] [Simplified Collect][Categories] Add delete functionality

### ISSUE THREAD

**Issue by luacmartins** [Expensify/App#38316]
Implement [this section](https://docs.google.com/document/d/1gk3xqOs7epMbUrSSiX8K7YcqfPLVgqEos0sf-D-GMDA/edit#bookmark=id.hn2v8ktjvjvl) of the design doc

cc @ArekChr 
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~011ae2fed7ed60f4ac</li>
        <li>Upwork Job ID: 1768330723656794112</li>
        <li>Last Price Increase: 2024-03-14</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @kevinksullivan</details>

**Comment by ArekChr** [comment]
Hey @luacmartins please assign me

**Comment by situchan** [comment]
This is new feature. I think regression test will be added when simplified collect is done. Not needed for this specific issue.

**Comment by luacmartins** [comment]
Correct, no need for tests. Bump @kevinksullivan for payment.

**Comment by kevinksullivan** [comment]
@situchan offer sent, lmk when you accept

**Comment by luacmartins** [comment]
Still pending payment

**Comment by luacmartins** [comment]
Just pending payment

**Comment by luacmartins** [comment]
Bump for payment @kevinksullivan

**Comment by luacmartins** [comment]
Bump for payment @kevinksullivan

**Comment by luacmartins** [comment]
Bump @kevinksullivan for payment

**Comment by kevinksullivan** [comment]
@situchan can you accept the offer? It's still pending. 

<img width="1295" alt="image" src="https://github.com/Expensify/App/assets/22508290/3ab0c090-40b4-49f7-a96e-4a17dea3779b">

**Comment by kevinksullivan** [comment]
Looks like @situchan is OOO. Tagged here as well.

https://expensify.slack.com/archives/C01GTK53T8Q/p1714510720258829

**Comment by kevinksullivan** [comment]
Setting to weekly since we are waiting on @situchan

**Comment by kevinksullivan** [comment]
Still waiting on @situchan

**Comment by situchan** [comment]
@kevinksullivan sorry I am back. can you please send new offer?

**Comment by kevinksullivan** [comment]
Welcome back!

**Comment by kevinksullivan** [comment]
@situchan did this offer work?

https://www.upwork.com/jobs/~011ae2fed7ed60f4ac

**Comment by kevinksullivan** [comment]
all set
